identity-service/src/config.js

```diff
@@ -235,12 +241,24 @@ const config = convict({
     env: 'minimumBalance',
     default: null
   },
+  minimumRelayerBalance: {
```

Comment:
@dmanjunath why make these configs and not route parameters? seems like this is an intermediate step to doing that